crates/ruff_python_ast/src/imports.rs

```diff
@@ -104,21 +106,27 @@ impl FutureImport for AnyImport<'_> {
 /// A representation of a module reference in an import statement.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ModuleImport {
-    module: String,
+    pub module: Arc<str>,
     location: Location,
     end_location: Location,
 }
 
 impl ModuleImport {
     pub fn new(module: String, location: Location, end_location: Location) -> Self {
         Self {
-            module,
+            module: Arc::from(module),
             location,
             end_location,
         }
     }
 }
 
+impl AsRef<ModuleImport> for ModuleImport {
```

Comment:
It's unclear to me why it's necessary to implement `AsRef<ModuleImport`. Could you give me an example of how it is used?

Reply:
`imports.module_to_imports[module_name][0].as_ref().into(),`
Line 125 of `cylic_import.rs`
Could I have avoided it?
`imports.module_to_imports[module_name]` yields a `ModuleImport`, I made `impl From<&ModuleImport> for Range` only so I need a reference to a `ModuleImport`.

Reply:
You can write `(&imports.module_to_imports[module_name][0]).into()`.

Reply:
@andersk, thanks! I thought I tried this, but I must've been mistaken. Thanks for the review/bug finding btw.